syntax/verilog_systemverilog.vim

```diff
@@ -289,7 +289,7 @@ endif
 
 if index(s:verilog_syntax_fold, "instance") >= 0 || index(s:verilog_syntax_fold, "all") >= 0
     syn region verilogFold
-        \ start="^\s*\w\+\(\s*#\s*(\|\s\+\w\+\s*(\)\(.*)\s*\w\+\s*;\)\@!"
+        \ start="^\s*\(\<\(task\|function\|checker\)\>\)\@!\w\+\(\s*#\s*(\(.*)\s*\w\+\s*;\)\@!\|\s\+\(\<if\>\)\@!\w\+\s*(\)"
```

Comment:
I think I recall reading this help section sometime ago, but I had completely forgotten about it!
Can't thank you enough!